package.json

```diff
@@ -129,7 +129,7 @@
     "blanket": "1.1.6",
     "chai": "2.0.0",
     "deep-freeze": "0.0.1",
-    "enzyme": "1.1.0",
+    "enzyme": "2.2.0",
```

Comment:
That's the plan, but this PR is more of a playground than production line. Thanks for the review!